### PR TITLE
Winner of game

### PR DESCRIPTION
**Pull request by grxver7**
Can someone finish this.
Calculate the winner and then put his name or his icon in the label for first, second, third

### CODE REVIEW

**Comment by KevinBasener** [comment]
Added winner screen in the end

File: main/main.gd [hunk 1]
```diff
@@ -55,6 +54,14 @@ func start_game(num_rounds, num_players):
 				break
 			await player_turn(player)
 
+	State.players.sort_custom(func(player1, player2): 
+			return player1.riesen + player1.riesen < player2.riesen + player2.patente
```
Comment (hfxbse) [inline main/main.gd]: The calculation is off, as the first sum is not the same as the second one. Also, does a patent have the same value as the money currency?

**Review by hfxbse — CHANGES_REQUESTED** [review]

File: main/main.gd (hunk 1 above)
Comment (ubeF) [reply main/main.gd]: As far as i understand it, the value is the same. Half of the value is simply lost when buying patente

Comment (KevinBasener) [reply main/main.gd]: Oh a typo

Comment (KevinBasener) [reply main/main.gd]: Changed it to
`State.players.sort_custom(func(player1, player2): 
			return player1.riesen + player1.patente < player2.riesen + player2.patente
	)
`

**Review by ubeF — APPROVED** [review]

**Review by hfxbse — APPROVED** [review]

**Pull request merged**